trajopt_utils/include/trajopt_utils/eigen_conversions.hpp

```diff
@@ -5,7 +5,7 @@
 
 namespace util
 {
-inline std::vector<double> toDblVec(const Eigen::VectorXd& x)
+inline std::vector<double> toDblVec(const Eigen::MatrixXd& x)
```

Comment:
I wanted to just check if this was intended. Note that Eigen by default is column major.

Reply:
I thought that was required for some reason, but now I don't see where. I'll change it back.

Reply:
I found it again. Constraints have to return a vector. In the case of a joint constraint over multiple timesteps, we have a constraint matrix that can be timesteps x dof. By changing it to Matrix, it should unwrap the matrix into a std::vector that the optimizer can evaluate that is timesteps*dof long.

As far as being column major, this seems to work. Should we make it RowMajor for performance?

Reply:
I am not sure on the RowMajor versus Column major. I believe RowMajor is used for Trajectory type but not sure exactly why.

On another note I would recommend using Eigen::Ref when possible. `const Eigen::Ref<Eigen::MatrixXd>& x` see [here](https://eigen.tuxfamily.org/dox/classEigen_1_1Ref.html).